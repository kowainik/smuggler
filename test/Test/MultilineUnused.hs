module Test.MultilineUnused where

import Data.Bool (bool)
import Data.List (break, cycle, drop, dropWhile, filter, genericDrop, genericLength,
                  genericReplicate, genericSplitAt, genericTake, group, inits, intercalate,
                  intersperse, isPrefixOf, iterate, map, permutations, repeat, replicate, reverse,
                  scanl, scanr, sort, sortBy, sortOn, splitAt, subsequences, tails, take, takeWhile,
                  transpose, unfoldr, unzip, unzip3, zip, zip3, zipWith, (++))

foo = bool
