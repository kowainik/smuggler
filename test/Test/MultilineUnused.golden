module Test.MultilineUnused where

import Data.Bool (bool)

foo = bool
